README.rst

```diff
@@ -6,24 +6,35 @@
 redturtle.agidtheme
 ==============================================================================
 
+Il primo tema Plone conforme alle linee guida `AgID`__.
+
+__ http://www.agid.gov.it/tags/linee-guida-design-pa
+
+Questo tema si basa sulla versione 2017.1 delle linee guida.
+  
+|
+
 This is the first Plone theme for `AgID`__ guidelines.
 
 __ http://www.agid.gov.it/tags/linee-guida-design-pa
 
 It is built on guidelines' version 2017.1.
 
-Documentation
--------------
+This README is written in italian language because it's meant for Italian Public Administrations' websites.
```

Comment:
Se non vado errato qui conviene mantenere la dicitura all'italiana "Amministrazione Pubblica" anche al plurale, quindi `Italian Public Administration websites`. Sempre se ricordo bene, ma correggetemi se sbaglio, il genitivo sassone si usa solo per gli esseri viventi.

Reply:
Si, avevo qualche dubbio in effetti :white_check_mark: